libs/dao/src/main/java/com/akto/dto/ApiCollection.java

```diff
@@ -35,6 +42,19 @@ public ApiCollection(int id, String name, int startTs, Set<String> urls, String
         this.urls = urls;
         this.hostName = hostName;
         this.vxlanId = vxlanId;
+        this.redact = false;
+        this.sampleCollectionsDropped = true;
+    }
+
+    public ApiCollection(int id, String name, int startTs, Set<String> urls, String hostName, int vxlanId, boolean redact, boolean sampleCollectionsDropped) {
+        this.id = id;
+        this.name = name;
+        this.startTs = startTs;
+        this.urls = urls;
+        this.hostName = hostName;
+        this.vxlanId = vxlanId;
+        this.redact = redact;
+        this.sampleCollectionsDropped = sampleCollectionsDropped;
```

Comment:
Please use a single constructor here. Lets not add a new constructor

Reply:
Fixed.